tests/foreman/api/test_subnet.py

```diff
@@ -497,6 +496,34 @@ def test_positive_list_parameters(self):
         :expectedresults: The satellite should display all the subnet
             parameters
         """
+        parameter = [{'name': gen_string('alpha'),
+                      'value': gen_string('alpha')}]
+        org = entities.Organization().create()
+        loc = entities.Location(organization=[org]).create()
+        org_subnet = entities.Subnet(
+            location=[loc],
+            organization=[org],
+            ipam=u'DHCP',
+            vlanid=gen_string('numeric', 3),
+            subnet_parameters_attributes=parameter,
+        ).create()
+        self.assertEqual(org_subnet.subnet_parameters_attributes[0]['name'],
+                         parameter[0]['name'])
+        self.assertEqual(org_subnet.subnet_parameters_attributes[0]['value'],
+                         parameter[0]['value'])
+        sub_param = entities.Parameter(
+            name=gen_string('alpha'),
+            subnet=org_subnet.id,
+            value=gen_string('alpha')
+        ).create()
+        sub = entities.Subnet(id=org_subnet.id).read()
```

Comment:
As far as I can tell, this is the same `org_subnet` as before (only with new parameter added). Since you never refer to `org_subnet` after this point, maybe just re-use variable name?

Reply:
Done.